controls/V-72219.rb

```diff
@@ -1,7 +1,65 @@
 # encoding: utf-8
 #
 
-# TODO we need to write tests on this.
+firewalld_services_deny = input(
+  'firewalld_services_deny',
+  value: {
+    # Examples (zone:[services])
+    # "public"=>['ssh', 'icmp'],
+    # "dmz"=>['http']   
+  },
+  description: "Services that firewalld should be configured to deny."
+)
+
+firewalld_hosts_deny = input(
+  'firewalld_hosts_deny',
+  value: [
+    # Example
+    # 'rule family="ipv4" source address="104.56.21.1/24" accept]'
+  ],
+  description: "Hosts that firewalld should be configured to deny."
+)
+
+firewalld_ports_deny = input(
+  'firewalld_ports_deny',
+  value: {
+    # Examples (zone:[ports])
+    #"public"=>['12345/tcp', '23456/tcp'],
+    # "dmz"=>['80/http', '22/ssh']
+  },
+  description: "Ports that firewalld should be configured to deny."
+)
+
+default_firewalld_zone = input(
+  'default_firewalld_zone',
+  value: "public",
```

Comment:
Why does this need to be your default zone?

Reply:
What would you prefer for the default zone?

Reply:
@djhaynes It should just be discovered shouldn't it? The name is irrelevant as long as the settings are correct so I don't think that it's something that needs to be checked at all.

Reply:
I removed the input and the check for the default firewall zone.